doc/en/fcrontab.1.sgml

```diff
@@ -53,22 +53,26 @@ A copy of the license is included in gfdl.sgml.
 	    linkend="fcron.8">&fcron;(8)</link> daemon. As &fcron; internally
 	    uses a non-human readable format (this is needed because &fcron;
 	    saves more informations than the user gives, for example the time
-	    and date of next execution), the user cannot edit directly his
-	    &fcrontabf; (the one used by &fcron;).</para> <para>When a user
+	    and date of next execution), the user cannot edit his &fcrontabf;
```

Comment:
while we are at it, do you mind replacing "his" by "their" ?

Reply:
sure, +1 for diversity :)